Documentation/ceph-nfs-crd.md

```diff
@@ -73,6 +73,8 @@ spec:
 * `pool`: The pool where ganesha recovery backend and supplemental configuration objects will be stored
 * `namespace`: The namespace in `pool` where ganesha recovery backend and supplemental configuration objects will be stored
 
+> **NOTE**: Don't use EC pool for this purpose because ganesha uses omap in the recovery objects and grace db. EC pool do not support omap.
```

Comment:
```suggestion
> **NOTE**: Don't use EC pools for NFS because ganesha uses omap in the recovery objects and grace db. EC pools do not support omap.
```

Reply:
Done!